sdk/src/main/java/co/omise/android/ui/AuthorizingPaymentActivity.kt

```diff
@@ -265,7 +267,7 @@ class AuthorizingPaymentActivity : AppCompatActivity() {
         val resultIntent = Intent().apply {
             putExtra(EXTRA_AUTHORIZING_PAYMENT_RESULT, Failure(throwable))
         }
-        if (arrayOf("Challenge protocol error", "Challenge runtime error", "3DS2 initialization failed").contains(throwable.message)) {
+        if (arrayOf(ChallengeStatus.PROTOCOL_ERROR.value, ChallengeStatus.RUNTIME_ERROR.value, OmiseSDKError.THREE_DS2_INITIALIZATION_FAILED.value).contains(throwable.message)) {
```

Comment:
Is there a way to make this line shorter?

Reply:
Formatted